records_versioning/services.py

```
@@ -0,0 +1,249 @@
+import threading
+import uuid
+from typing import Any, Dict, Optional
```

Ruff (0.12.2)

`typing.Dict` is deprecated, use `dict` instead

(UP035)